api/python/cellxgene_census/src/cellxgene_census/experimental/pp/_online.py

```diff
@@ -130,13 +160,22 @@ def _mbomv_update_by_batch(
 
 
 @numba.jit(
-    numba.void(
-        numba.types.Array(numba.int64, 1, "C", readonly=True),
-        numba.types.Array(numba.float32, 1, "C", readonly=True),
-        numba.int32[:, :],
-        numba.float64[:, :],
-        numba.float64[:, :],
-    ),
+    [
```

Comment:
This is required because previously `test_meanvar` was hitting the batched case even for n_batches=1. With the changes, it's now hitting this code and the type constraints didn't match the test case where `var_vec` is an int32.

Reply:
really good catch!!!  Thank you.

Reply:
actually, don't we have the same issue on all of the routines that accept coordinates - they are all wired to only accept int64 (I did this originally because I assumed they would all be soma_joinids, but you are doing the right thing by assumig they could be int32s in some cases).  so I see the same issue in _accum_clipped_counts and _accum_clipped_counts_by_batch.

Reply:
I think that's because `CountAccumulator` lacks an explicit test, as you pointed out. I will add it and we'll likely need to extend this change there.